packages/truffle/test/sources/toyProject/contracts/Migrations.sol

```diff
@@ -0,0 +1,19 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.4.22 <0.9.0;
+
+contract Migrations {
+  address public owner = msg.sender;
+  uint public last_completed_migration;
+
+  modifier restricted() {
```

Comment:
Does it make sense to have a fixture project with and without a Migrations contract?